lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.prototype_inputs.json

```diff
@@ -1050,7 +1050,7 @@
       "exterior_fuel_equipment1_power": 244443.956043956,
       "exterior_fuel_equipment1_schedule": "Large Office BLDG ELEVATORS",
       "exterior_fuel_equipment2_power": 1270.8,
-      "exterior_fuel_equipment2_schedule": "OfficeLarge ELEV_LIGHT_FAN_SCH_ADD_DF",
+      "exterior_fuel_equipment2_schedule": null,
```

Comment:
@leijerry888 - Is that in line with the elevator changes you've made?

Reply:
@lymereJ 
No. For 2004 large office, I changed from 

      "exterior_fuel_equipment2_power": 1270.8,
      "exterior_fuel_equipment2_schedule": "OfficeLarge ELEV_LIGHT_FAN_SCH_ADD_DF",

to

      "exterior_fuel_equipment2_power": 1942.8,
      "exterior_fuel_equipment2_schedule": "OfficeLarge ELEV_LIGHT_FAN_SCH_24_7",

see #844

Reply:
This is for the detailed large office building, so we're good.